package-build.el

```diff
@@ -501,17 +497,18 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
         (package-build--princ-checkout repo dir)
         (package-build--run-process nil "git" "clone" repo dir)))
       (if package-build-stable
-          (let* ((bound (goto-char (point-max)))
-                 (tag-version (and (package-build--run-process dir "git" "tag")
-                                   (or (package-build--find-tag-version-newest
-                                        "^\\(?:v[.-]?\\)?\\([0-9]+[^ \t\n]*\\)$" bound)
-                                       (error
-                                        "No valid stable versions found for %s"
-                                        name)))))
-            ;; Using reset --hard here to comply with what's used for
-            ;; unstable, but maybe this should be a checkout?
-            (package-build--update-git-to-ref dir (concat "tags/" (car tag-version)))
-            (cadr tag-version))
+        (let* ( (bound (goto-char (point-max)))
+                (regexp (or (plist-get (package-build--config name) :version)
```

Comment:
don't need `package-build--config` here because the `config` variable should already contain the config hash. 

thus you can use `(plist-get config :version)`.